docs/book/src/cronjob-tutorial/testdata/project/Makefile

```diff
@@ -152,10 +152,10 @@ $(LOCALBIN):
 
 ## Tool Binaries
 KUBECTL ?= kubectl
-KUSTOMIZE ?= $(LOCALBIN)/kustomize-$(KUSTOMIZE_VERSION)
-CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen-$(CONTROLLER_TOOLS_VERSION)
-ENVTEST ?= $(LOCALBIN)/setup-envtest-$(ENVTEST_VERSION)
-GOLANGCI_LINT = $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
+KUSTOMIZE ?= $(LOCALBIN)/kustomize
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+GOLANGCI_LINT = $(LOCALBIN)/golangci-lint
```

Comment:
While this change might be helpful for the way you are using Tilt, it's not clear how many other users would benefit from this adjustment. Could you provide more context or examples of how this change would improve the experience for the majority of users?

Also, what is Tilt? Who uses it? When it is used?

Reply:
Tilt is a tool that helps to run the development cycle of an application in a kubernetes cluster. It's _very_ useful when developing operators/controllers since it makes it very easy to run your app in a cluster for testing. Especially webhooks can be debugged a lot easier using it. https://tilt.dev

A common use-case with tilt is to run kustomize to build the manifests from the config folder and then use them for the tilt deployment.

cluster-api is a very heavy user of tilt for example. They actually do the same thing, but with an external script that's called from the Makefile: https://github.com/kubernetes-sigs/cluster-api/blob/main/scripts/go_install.sh#L45